src/utils/index.js

```diff
@@ -9,7 +25,20 @@ export const setupDocsearch = () => {
     apiKey: "f53a424573adab20d04faa2db150c349",
     indexName: "livechatinc",
     inputSelector: "#search",
-    debug: false
+    debug: false,
+    handleSelected: function (input, event, suggestion, datasetNumber, context) {
+      if (
+        context.selectionMethod === "click" ||
+        context.selectionMethod === "enterKey"
+      ) {
+        logAmplitudeEvent("User selected suggestion and input", {
```

Comment:
I'd suggest using a standardized pattern for naming events. For instance: 
- always use present tense (it's shorter and a bit easier to read) or always use past tense (longer, but might be less confusing),
- don't include the subject (it's already defined implicitly by the way it's displayed in the analytics panels),
- stick to the same convention throughout the entire project (it's easier to guess the name of the event).

There are multiple approaches to pick from: 

- https://help.amplitude.com/hc/en-us/articles/115000465251#how-should-i-name-my-events
- https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/
- https://supermetrics.com/blog/naming-conventions-google-analytics